src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/translations/messages.en.yml

```diff
@@ -16,7 +16,7 @@ entity.delete.request.ticket.description: "h2. Details
 *Applicant name*: %applicant_name%
 *Applicant email*: %applicant_email%.
 *Entity name (en)*: %entity_name%."
-
+entity.delete.flash.message: "Your entity is deleted."
```

Comment:
Perhaps, for the sake of consistency: rename the translation key to read: `entity.delete.flash.success`.

Reply:
Agree.